### PR TITLE
Added a Pull request template

### PR DESCRIPTION
**Pull request by John-pels**

### CODE REVIEW

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Description
+
+Fork and clone the repository,create a branch for your whatever you wanna work on, then add, commit and push to your forked repository and create a PR. Do ensure there is no conflict in your PR. Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+
+Fixes # (issue)
+
+## Type of change
```
Comment (SEUNAGBEYE) [inline docs/PULL_REQUEST_TEMPLATE.md]: Why do you have `##` here? You do know it's gonna make this title smaller right?

**Comment by SEUNAGBEYE** [comment]
Avoid using `past tense` for your PR sentence. Instead, use present tense like e.g `Add Pull request template`

**Comment by John-pels** [comment]
Okay, noted.

File: docs/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (John-pels) [reply docs/PULL_REQUEST_TEMPLATE.md]: Okay, I will remove it now

**Pull request closed without merging**